public/plugin.tsx

```diff
@@ -101,7 +101,12 @@ export const [
   getRenderAccelerationDetailsFlyout,
   setRenderAccelerationDetailsFlyout,
 ] = createGetterSetter<
-  (index: string, acceleration: CachedAcceleration, dataSourceName: string) => void
+  (
```

Comment:
we should also update flyout params in types: https://github.com/opensearch-project/dashboards-observability/blob/main/public/types.ts

Reply:
fixed.